keps/sig-node/2400-node-swap/README.md

```diff
@@ -587,13 +591,19 @@ Try to be as paranoid as possible - e.g., what if some components will restart
 mid-rollout?
 -->
 
+If a new node with swap memory fails to come online, it will not impact any
+running components.
```

Comment:
If someone does an in-place upgrade on a node (stopping kubelet, starting a new kubelet on the same server), can that fail? How?

If it _could_ fail then an upgrade might for example take out the nodes where the control plane ought to be running as static pods.

Reply:
The in-place upgrade would not fail unless swap access was added while the node was still online. Normally turning swap on and off at runtime isn't considered best practice for a production environment, I'd expect a node to be reimaged and rebooted, but I can mention it.